dlt/destinations/job_client_impl.py

```diff
@@ -58,6 +59,38 @@ def is_sql_job(file_path: str) -> bool:
         return os.path.splitext(file_path)[1][1:] == "sql"
 
 
+class CopyFileLoadJob(LoadJob, FollowupJob):
+    def __init__(self, table: TTableSchema, file_path: str, sql_client: SqlClientBase[Any], forward_staging_credentials: bool = True, staging_config: Optional[DestinationClientStagingConfiguration] = None) -> None:
```

Comment:
looks like we do not need `forward_staging_credentials`. we forward when staging credentials are present and that's it!

Reply:
removed!